woocommerce/woocommerce-gutenberg-products-block
# Chip component stories for storybook

Refactoring the stories for the Chip component for compatibility with the new Storybook.


Fixes #5251 

How to test the changes in this Pull Request:

1. Open up storybook with `npm run storybook`
2. You should be able to see the `Chip` and `RemovableChip` components.
3. Play around!
